src/api/pipelineRuns.js

```diff
@@ -121,3 +121,12 @@ export function rerunPipelineRun(pipelineRun) {
   const uri = getTektonAPI('pipelineruns', { namespace });
   return post(uri, payload).then(({ body }) => body);
 }
+
+export function startPipelineRun(pipelineRun) {
+  const { name, namespace } = pipelineRun.metadata;
+
+  const payload = [{ op: 'remove', path: '/spec/status' }];
+
+  const uri = getTektonAPI('pipelineruns', { name, namespace });
+  return patch(uri, payload).then(({ body }) => body);
```

Comment:
We've just recently (i.e. Tuesday) updated the API proxy in the backend and it's now correctly handling the request/response headers, in particular the `Accept` / `Content-Type` headers.

It looks like our original implementation of `patch` wasn't quite right. In api/comms.js `getPatchHeaders` you'll need to update the value of the `Accept` header to `application/json` to resolve the 406 error.